### PR TITLE
Add validation for JSON files with NodeJS

### LINKED ISSUE #39: Validate JSON files using jq
We should validate all PRs using jq or a similar tool with Travis. This prevents the store from breaking when the JSON is merged but invalid.

### PR DESCRIPTION
**Pull request by ivanayov**
This change allows pull requests to be validated using Travis and jq package.
 At this stage only JSON syntax validation is provided

Signed-off-by: Ivana Yovcheva <iyovcheva@vmware.com>

Resolves #39

### CODE REVIEW

**Comment by ivanayov** [comment]
Thanks for the notice @alexellis 

I associated jq with javascript->nodejs.
It's now updated.

Tested locally

**Comment by ivanayov** [comment]
I committed an error in store.json to test. Will remove the commit.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+sudo: false
+
+install:
+  - sudo apt-get install jq
```
Comment (alexellis) [inline .travis.yml]: Is `sudo` needed here? (I'm not sure)

Comment (ivanayov) [reply .travis.yml]: Yes, it's from the docs. I just tested without sudo and the build failed with Permissions denied ([#L440](https://travis-ci.org/openfaas/store/builds/422164035#L440))

**Pull request merged**